package.json

```diff
@@ -9,7 +9,8 @@
     "author": "Corentin Girard",
     "license": "ISC",
     "dependencies": {
-        "brackets-manager": "^1.3.9"
+        "brackets-manager": "^1.3.9",
+        "rfdc": "^1.4.1"
```

Comment:
Why has this dependency been added?

Reply:
the package is imported in brackets-memory-db/index.ts the original file I edited and has been used there before my change. Therefore I decided to add it.

Reply:
Thanks for adding it, this must have been an oversight